step.yml

```diff
@@ -42,6 +42,14 @@ inputs:
       is_expand: true
       is_required: true
       value_options: []
+  - jira_project:
```

Comment:
Is it the JIRA project key ? If so maybe we should make it explicit in the summary / description ?